task/09-functions-n-closures-tasks.js

```diff
@@ -104,7 +113,17 @@ function memoize(func) {
  * retryer() => 2
  */
 function retry(func, attempts) {
-    throw new Error('Not implemented');
+    return function TryFunc(){
+        while (attempts>0){
```

Comment:
You have 2 exit conditions. you need only 1 in the loop definition. Also use for-loop
![Uploading image.png…]()